tools/online_editor/src/index.ts

```diff
@@ -112,21 +102,46 @@ function create_style_menu(editor: EditorWidget): Menu {
     return menu;
 }
 
-function create_open_menu(editor: EditorWidget): Menu {
+function create_settings_menu(): Menu {
+    const menu = new Menu({ commands });
+    menu.title.label = "Change Settings";
```

Comment:
My preference is just "Settings" - because the whole point of the action behind the menu item is to ... change them :-)

```suggestion
    menu.title.label = "Settings";
```